test/preprocessing/readvtk/vtk2trixi.jl

```diff
@@ -0,0 +1,77 @@
+@testset verbose=true "vtk2trixi" begin
+    output_directory = "test/preprocessing/readvtk"
+
+    Random.seed!(1)
+    saved_ic = InitialCondition(; coordinates=rand(2, 12), velocity=rand(2, 12),
+                                density=1.0, mass=2.0,
+                                pressure=3.0)
+
+    # `InitialCondition`-Files
+    trixi2vtk(saved_ic; output_directory,
+              filename="test_initial_condition")
+
+    loaded_ic = vtk2trixi(joinpath("test/preprocessing/readvtk",
```

Comment:
```suggestion
    test_ic = vtk2trixi(joinpath("test/preprocessing/readvtk",
```
and please use `joinpath`